src/support/parser.ts

```diff
@@ -37,7 +37,9 @@ export const setParserBinaryPath = (context: vscode.ExtensionContext) => {
 
 const downloadBinary = async (context: vscode.ExtensionContext) => {
     const binaryVersion = "0.1.6";
-    const filename = `php-parser-${binaryVersion}`;
+    const osPlatform = os.platform();
+    const osArch = os.arch();
```

Comment:
we need to ensure that the binary return has x64 and arm64 as `os.arch()` returns those possible values.

 'arm' , 'arm64' , 'ia32' , 'loong64' , 'mips' , 'mipsel' , 'ppc' , 'ppc64' , 'riscv64' , 's390' , 's390x' , and 'x64' 
which is different from https://github.com/SamuelNitsche/vs-code-php-parser-cli/releases/tag/0.1.9 

@joetannenbaum

Reply:
Yup! Thank you. Working on integrating this now.

Reply:
Is this related to the 'php-parser-0.1.6: Exec format error' preventing the extension from running?

Reply:
Yes